src/cvmix/gotm_cvmix.F90

```diff
@@ -390,21 +392,27 @@ subroutine init_cvmix_yaml(branch)
    call leaf%get(kpp_use_noDGat1, 'use_noDGat1',                       &
       'zero gradient of the shape function at OBL', default=.true.)
    call leaf%get(kpp_match_technique, 'match_technique',               &
-      'matching technique with the ocean interior',                    &
-      default=1, options=(/                                            &
-      option(1, 'SimpleShapes'),                                       &
-      option(2, 'MatchGradient'),                                      &
-      option(3, 'MatchBoth'),                                          &
-      option(4, 'ParabolicNonLocal')/))
+      'matching technique of shape functions with the ocean interior', &
+      default=CVMIX_MATCH_SIMPLE, options=(/                           &
+      option(CVMIX_MATCH_SIMPLE, 'simple shapes', 'simple'),           &
+      option(CVMIX_MATCH_GRADIENT, 'gradient term', 'gradient'),       &
+      option(CVMIX_MATCH_BOTH,                                         &
+             'both gradient and nonlocal terms', 'both'),              &
+      option(CVMIX_MATCH_PARABOLIC,                                    &
+             'parabolic nonlocal term', 'parabolic')/))
    call leaf%get(kpp_bulk_Ri_interp_type, 'bulk_Ri_interp_type',       &
       'interpolation type for the bulk Richardson number',             &
-      default=2, options=(/                                            &
-      option(1, 'linear'), option(2, 'quadratic'), option(3, 'cubic')/))
+      default=CVMIX_INTERP_QUADRATIC, options=(/                       &
+      option(CVMIX_INTERP_LINEAR, 'linear', 'linear'),                 &
+      option(CVMIX_INTERP_QUADRATIC, 'quadratic', 'quadratic'),        &
+      option(CVMIX_INTERP_CUBIC, 'cubic', 'cubic')/))
    call leaf%get(kpp_OBL_interp_type, 'OBL_interp_type',               &
       'interpolation type for diffusivity and viscosity at OBL',       &
-      default=4, options=(/                                            &
-      option(1, 'linear'), option(2, 'quadratic'),                     &
-      option(3, 'cubic'), option(4, 'LMD94')/))
+      default=CVMIX_INTERP_LMD94, options=(/                           &
+      option(CVMIX_INTERP_LINEAR, 'linear', 'linear'),                 &
+      option(CVMIX_INTERP_QUADRATIC, 'quadratic', 'quadratic'),        &
+      option(CVMIX_INTERP_CUBIC, 'cubic', 'cubic'),                    &
+      option(CVMIX_INTERP_LMD94, 'LMD94', 'LMD94')/))
```

Comment:
Could the first LMD94 be replaced by a more descriptive string as used elsewhere, e.g., first author + year?

Reply:
@jornbr Done.